frontend/packages/data-portal/app/constants/annotationInfoLinks.ts

```diff
@@ -0,0 +1,6 @@
+export const GO = 'https://amigo.geneontology.org/amigo/term/'
+export const UNIPROTKB = 'https://www.uniprot.org/uniprotkb/'
+
+export const GO_PATTERN = /GO:[0-9]{7}/
```

Comment:
curious if these regexes need to be so specific? What if we just matched "GO:" and "UniProtKB:"?

Reply:
These are the same regexes on the backend that ensure the values are correct, so I thought I'd just use the same here. Based on: https://bioregistry.io/registry/go and https://www.uniprot.org/help/accession_numbers